style.css

```diff
@@ -1,4 +1,395 @@
+@media only screen and (min-width: 480px) {
```

Comment:
- [ ] Please kindly make sure you use the right `min-width`, as required in the project the `min-width` should be in the range of `768px` or wider :upside_down_face: :upside_down_face: 
- [ ] Your Desktop should look at the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) that you chose.
Your design

![Screenshot from 2022-07-06 13-49-01](https://user-images.githubusercontent.com/74501523/177545110-81fe1d06-41f6-4451-8bec-a37992c2dacd.png)

Figma design

![Screenshot from 2022-07-06 13-49-33](https://user-images.githubusercontent.com/74501523/177545201-94f83668-89d5-432d-bbac-981b41795dcd.png)